package.json

```diff
@@ -25,5 +25,14 @@
     "eslint": "^8.2.0",
     "eslint-plugin-node-core": "iansu/eslint-plugin-node-core",
     "tape": "^5.2.2"
+  },
+  "files": [
+    "index.js",
+    "primordials.js"
+  ],
+  "eslint": {
+    "env": {
+      "es6": true
+    }
   }
```

Comment:
this should be in `.eslintrc`, and not in package.json at all - both because we already have one, and also because config inline in package.json unavoidably inflates package size.